ee/scripts/schema/db/rollback_dbs/postgresql/1.20.0/1.20.0.sql

```diff
@@ -28,6 +28,11 @@ DROP SCHEMA or_cache CASCADE;
 ALTER TABLE IF EXISTS public.tenants
     DROP COLUMN IF EXISTS scope_state;
 
+DROP TABLE IF EXISTS spots;
+DROP TABLE IF EXISTS spots_keys;
+DROP TABLE IF EXISTS spots_streams;
+DROP TABLE IF EXISTS spots_tasks;
```

Comment:
This won't work, as spots is at top table of the other tables, you have to drop childs first then drop parent